applications/odyssey_momentum.md

```diff
@@ -0,0 +1,278 @@
+  
+- Momentum is using Kusama (and optionally Polkadot) as its primary digital public infrastructure
+- Momentum will be launched as a parachain on Kusama (and optionally Polkadot in a later stage)
+- Momentum has an extensive community that actively participates in building Momentum. 200 is an estimate of active contributors based on GitHub commits, feedback during sessions and idea generation. Our community is over 7,000 people strong being a mix of web3 and non-web3 participants.
+- Momentum provides a new ecosystem with business cases for builder teams, individual developers, hosting companies, and tech- and non-tech entrepreneurs.
+- Momentum provides a bridge between the WEB2 and WEB3 community.
+
+
+What makes us different: 
+We see the metaverse as a new social communications medium and the potential driver for mainstream adoption of web3 technologies. The potential is designing and launching entirely new user experiences, rather than doing the web2/2d consumer experience and slapping a 3D background on it (a.k.a. a recorded concert in Fortnite that you can watch/consume or selling land). When unlocking the dotsama/ next generation of on-chain applications we enable consumers to become co-creators.
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+Odyssey  has over 30 people working on Momentum and is still growing. Odyssey works in tracks with dedicated team members. The Token Track Team will be primarily responsible for building the Substrate Pallets. 
+
+OVERALL ARCHITECT: Anton Starikov (CTO)
+
+**TOKEN TRACK TEAM**   
+- TRACK LEAD: Dave Hoogendoorn
+- RUST DEVELOPER: Denis Cavalli 
+- RUST DEVELOPER: Raghuvar Vijayakumar
+- WEB3ANALYST: Tim Jansen  
+
+*All team members are solely dedicated to the token Track.*
+
+
+### Contact
+
+- **Contact Name:** Dave Hoogendoorn
+- **Contact Email:** dh@odyssey.org
+- **Website:** https://odyssey.org/
+
+### Legal Structure
+
+- **Registered Address:** Westerhaven 13-7, 9718AW Groningen, The Netherlands
+- **Registered Legal Entity:** Odyssey B.V. (CoC 66693640)
+
+### Team's experience
+
+Please describe the team's relevant experience. If your project involves development work, we would appreciate it if you singled out a few interesting projects or contributions made by team members in the past. For research-related grants, references to past publications and projects in a related domain are helpful.
+
+If anyone on your team has applied for a grant at the Web3 Foundation previously, please list the name of the project and legal entity here.
```

Comment:
```suggestion

```
Could you remove this template text?

Reply:
@keeganquigley Done

Reply:
Thanks @dave4odyssey see my comments above.